src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double RADIUS = 10;
+    private static final Color COLOR = Color.WHITE;
+
+    enum RandomFigure {
+        CIRCLE,
+        ISOSCELES_TRAPEZOID,
+        RECTANGLE,
+        RIGHT_TRIANGLE,
+        SQUARE
+    }
```

Comment:
why did you put it before the fields of the class?
IMO we don't really need this enum, just use integer values in switch